src/components/tl/mlx5/tl_mlx5.h

```diff
@@ -140,7 +146,11 @@ typedef struct ucc_tl_mlx5_team {
     ucc_tl_mlx5_alltoall_t         *a2a;
     ucc_topo_t                     *topo;
     ucc_ep_map_t                    ctx_map;
+    int                             local_mcast_ctx_ready;
     ucc_tl_mlx5_mcast_team_t       *mcast;
+    ucc_status_t                    local_status_array[UCC_TL_MLX5_FEATURES_COUNT];
+    ucc_status_t                    global_status_array[UCC_TL_MLX5_FEATURES_COUNT];
```

Comment:
imho having bitmask here is better, in that case you don't need to maintain UCC_TL_MLX5_FEATURES_COUNT. But I don't mind having it as array

Reply:
Thanks for the comment.  We will change it in future PRs